### PR TITLE
Shortcuts on Mangaki

### LINKED ISSUE #13: Raccourcis clavier
Pour les nouveaux utilisateurs qui n'ont pas de MAL ou souhaitent procéder à la notation manuelle, avec une large quantité d'animes vont trouver cela assez douleureux de le faire sans raccourcis clavier.

Donc, ça serait pratique d'intégrer un moyen de noter au clavier assez rapidement.

(Un vim mode dans les options de profiles ? :smile:)

### PR DESCRIPTION
**Pull request by RaitoBezarius**
Remaining work:

- ~Add some profile setting to enable / disable shortcuts.~
- ~Some refactor / DRY.~
- ~CSS to make cards in focus style more obvious (the default glowing blue is cool, but maybe not sufficient).~

### Cheatsheet

Screenshot

### Settings

Screenshot

~**Depends on #531.**~

Fix #13.

### CODE REVIEW

**Comment by RaitoBezarius** [comment]
To redo and trigger a click on the already created listener.

**Comment by jilljenn** [comment]
Il faut que ça introduise le minimum de modifications au code JS.

File: mangaki/mangaki/templates/base.html [hunk 1]
```diff
@@ -18,7 +18,7 @@
     <link rel="icon" href="/static/img/favicon.png">
     <link rel="apple-touch-icon" href="/static/img/touch-icon-iphone.png">
     <link rel="stylesheet" href="/static/css/bootstrap.min.css"/>
-    <link rel="stylesheet" href="/static/css/bootstrap-switch.min.css"/>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/css-toggle-switch@4/dist/toggle-switch.min.css">
```
[the remark this answers is not in the document]
Comment (jilljenn) [reply mangaki/mangaki/templates/base.html]: (Je me rends compte que je n'avais jamais envoyé cette review :X)

**Comment by jilljenn** [comment]
(Tain elle était en page 4.)

**Pull request merged**